### ISSUE TITLE
Improve styling of the block in editor and front-end

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-magnetic-poetry#3]
As commented in the readme “It sometimes looks a bit funny”.
Answer: Yes, it needs some better CSS.

**Comment by bobbingwide** [comment]
Try `display: flex`
See https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Relationship_of_Flexbox_to_Other_Layout_Methods

**Comment by bobbingwide** [comment]
I updated the readme to remove the "sometimes it looks funny" 
The examples on blocks.wp-a2z.org show how the block can be styled. 
Also https://herbmiller.me/oik-magnetic-poetry-v0-2-2/
shows text and background colours or gradient.

In v0.3.0 I made a change so that the font size was reflected in the editor.
But this doesn't seem to apply to the rendered version in the editor. 
This is because the block editor adds styling
```
.editor-styles-wrapper p {
font-size: var( --wp--preset--font-size--normal);
}
```
directly into the page. This takes precendence.

v0.3.0 is not the final solution therefore.
